drivers/edac/edac_ibecc.c

```diff
@@ -18,6 +18,12 @@
 #include <logging/log.h>
 LOG_MODULE_REGISTER(edac_ibecc, CONFIG_EDAC_LOG_LEVEL);
 
+#define DEVICE_NODE DT_NODELABEL(ibecc)
```

Comment:
Why is this using nodelabel vs compat?

Reply:
The main thing that bothers me with this approach is that you put special meaning to the node name, i.e. any node, regardless of what its compatible string, with `ibecc` as its name would match this driver, when in fact this driver is strictly for Intel-specific HW (as implied by the `compatible = "intel,ibecc";`. When at some point we start supporting IBECC hardware from other vendors this approach would break.

In fact, since this is an Intel-specific driver, perhaps we should have `intel` somewhere in the file name (rather than the current quite generic `edac_ibecc.c`?

Reply:
Added DT_DRV_COMPAT()